doc/content/modules/user-manual/pages/release-notes/2025.1.0.adoc

```diff
@@ -11,6 +11,7 @@
 == Improvements
 
 - `OccurrenceUsage#portionKind` is now `unsettable` and its default value is `null` in the SysMLv2 metamodel to conform to the specification.
+- Makes the importer more generic. It now reports more messages to understand the scope of what is imported and the errors encountered.
```

Comment:
Please also change the first sentence of this release note entry with the one I ask you to update in the changelog.
The second sentence is ok, may be just detail that the report is on the server side.